src/providers/toast.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { ToastController } from 'ionic-angular';
+
+@Injectable()
+export class ToastService {
+    private faveToast; 
+    constructor(private toast: ToastController) { } 
+    
+    favoriteToast(routeOrStop: string, isFave: boolean): void {
+        if (isFave){ 
```

Comment:
Noted: use string interpolation over concatenation and ternary statements when they fit on one line.